### PR TITLE
add window setAppDetails feature to modify windows user task in taskbar

### PR DESCRIPTION
**Pull request by fregie**
add window setAppDetails feature to modify windows user task
like this:
image

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -520,6 +520,17 @@ function windowCreateFinish(json) {
             url: url
         })
     })
+    if (typeof json.windowOptions.appDetails !== "undefined"){
```
Comment (asticode) [inline main.js]: Can you add the `windows only` logic here?

That would mean adding `process.platform === "win32"` in the `if` condition.

File: main.js [hunk 2]
```diff
@@ -520,6 +520,17 @@ function windowCreateFinish(json) {
...
+        elements[json.targetID].setThumbarButtons([]);
+        elements[json.targetID].setAppDetails({
```
Comment (asticode) [inline main.js]: You don't have to build a new object here, you can feed `json.windowOptions.appDetails` directly to `elements[json.targetID].setAppDetails`

**Review by asticode — CHANGES_REQUESTED** [review]
Cool idea.

Minor changes requested though.

**Comment by fregie** [comment]
Already modified as requested

**Comment by asticode** [comment]
Great! Is this still working as expected?

**Comment by fregie** [comment]
yes, it is, i tried

**Pull request merged**